aiida_pseudo/cli/params/options.py

```diff
@@ -7,18 +7,56 @@
 from aiida.cmdline.params.options import OverridableOption
 from .types import PseudoPotentialFamilyTypeParam
 
-__all__ = ('VERSION', 'FUNCTIONAL', 'PROTOCOL', 'TRACEBACK', 'FAMILY_TYPE', 'ARCHIVE_FORMAT')
+__all__ = (
+    'VERSION', 'FUNCTIONAL', 'REL', 'PROTOCOL', 'HINT', 'PSEUDO_FORMAT', 'TRACEBACK', 'FAMILY_TYPE', 'ARCHIVE_FORMAT'
+)
 
 VERSION = OverridableOption(
-    '-v', '--version', type=click.STRING, required=False, help='Select the version of the SSSP configuration.'
+    '-v',
+    '--version',
+    type=click.STRING,
+    required=False,
+    help='Select the version of the SSSP or PseudoDojo configuration.'
 )
 
 FUNCTIONAL = OverridableOption(
-    '-f', '--functional', type=click.STRING, required=False, help='Select the functional of the SSSP configuration.'
+    '-x',
+    '--functional',
+    type=click.STRING,
+    required=False,
+    help='Select the functional of the SSSP or PseudoDojo configuration.'
+)
+
+REL = OverridableOption(
```

Comment:
I'd really prefer this to be fully spelled out. Succinct just makes it less clear
```suggestion
RELATIVISTIC = OverridableOption(
```